magefile.go

```diff
@@ -125,6 +126,17 @@ func dryRunPackageRegistry() error {
 		return errors.Wrapf(err, "can't change directory to %s", buildDir)
 	}
 
+	// Creates a basic package-registry config which points to the packages.
+	config := `
+package_paths:
+- "../packages"
+`
+
+	err = ioutil.WriteFile("config.yml", []byte(config), 0644)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Why was this change needed as part of the version update?

Reply:
We got rid of the default public_dir in the registry. Should we introduce a default for package_paths?

Reply:
@ycombinator My hope is that in the future, we don't need the "-dry-run" of the registry anymore here and can just validate it with elastic-package.

Reply:
TBH I'm not super familiar with the package registry code or the motivations behind https://github.com/elastic/package-registry/pull/632 to have a solid opinion here. I see @mtojek reviewed that PR so I'll defer to his opinion here.

Reply:
@ycombinator With public_dir and package_paths we had 2 ways of configuring the same thing, so we god rid of the legacy option.

Reply:
It looks like a static file. Is there an option to keep it versioned in Git instead of creating dynamically?

Reply:
Any preference on where to put it. The main requirement is that we also run the registry from this directory.

Reply:
@mtojek I updated the PR and created the file in `testing`.